assets/partners/anthonydb/just_connect.py

```diff
@@ -1,4 +1,4 @@
-import sqlalchemy as sa
+import sqlalchemy as sa  # noqa: TID251
```

Comment:
@Kilo59 How come we are not using our compatibility modules here?  Just curious -- thanks.